internal/cmd/configssh.go

```diff
@@ -181,20 +195,6 @@ func makeSSHConfig(host, userName, envName, privateKeyFilepath string) string {
 `, envName, host, userName, envName, privateKeyFilepath)
 }
 
-func isSSHAvailable(ctx context.Context) bool {
-	ctx, cancel := context.WithTimeout(ctx, 3*time.Second)
-	defer cancel()
-
-	host, err := configuredHostname()
```

Comment:
I can't recall, can the `configuredHostname` func be removed as a result of this? May have been the only spot it was called